Makefile.core.mk

```diff
@@ -92,7 +92,7 @@ format-spelling:
 	@LC_ALL=C sort .spelling --ignore-case -o .spelling
 	@echo ".spelling file sorted."
 
-gen: tidy-go format-go update-gateway-version snips format-spelling
```

Comment:
Remove `update-gateway-version` target so the version isn't updated to a commit.